src/qt/bitcoingui.cpp

```diff
@@ -110,6 +110,7 @@ BitcoinGUI::BitcoinGUI(QWidget *parent):
 
     setGeometry(QStyle::alignedRect(Qt::LeftToRight,Qt::AlignCenter,QDesktopWidget().availableGeometry(this).size() * 0.6,QDesktopWidget().availableGeometry(this)));
 
+    QFontDatabase::addApplicationFont(":/fonts/Inter-Regular.ttf");
```

Comment:
This should use the resource alias and not the actual file name.

:/fonts/inter-regular

Reply:
As per the documentation it needs to be the filename.
https://doc.qt.io/qt-5/qfontdatabase.html#addApplicationFont